### PR TITLE
Add check class onlineMediaFile

### PR DESCRIPTION
**Pull request by cholletNovius**
Add check class onlineMediaFile to show média thumbnail in preview block.

### CODE REVIEW

File: views/common/content_class.view.php [hunk 1]
```diff
@@ -14,8 +14,16 @@
         <?php
         }
         break;
-
+        
+    case 'Novius\OnlineMediaFiles\Model_Media':
+        if (!empty($value)) {
```
Comment (Meroje) [inline views/common/content_class.view.php]: this `!empty` is not neccessary, $value _is_ an instance of Model_Media when you get here

**Comment by cholletNovius** [comment]
It's modified

**Comment by cholletNovius** [comment]
It's good or not ?

File: views/common/content_class.view.php [hunk 2]
```diff
@@ -14,8 +14,14 @@
         <?php
         }
         break;
-
+
```
Comment (jguyomard) [inline views/common/content_class.view.php]: trailing whitespaces ;)

File: views/common/content_class.view.php [hunk 3]
```diff
@@ -14,8 +14,14 @@
...
         }
         break;
-
+        
+    case 'Novius\OnlineMediaFiles\Model_Media':
+        ?>
+        <img src="<?= $value->thumbnail() ?>" style="width:100%">
+        <?php
+        break;
+
```
Comment (jguyomard) [inline views/common/content_class.view.php]: trailing whitespaces ;)

**Comment by cholletNovius** [comment]
ok

**Comment by cholletNovius** [comment]
up !

**Pull request merged**